caluma/caluma_workflow/filters.py

```diff
@@ -200,14 +203,12 @@ class WorkItemOrderSet(BaseFilterSet):
             "created_at",
             "modified_at",
             "closed_at",
-            "allow_all_forms",
             "description",
-            "is_archived",
-            "is_published",
             "name",
             "deadline",
             "status",
             "slug",
+            "task__is_archived",
```

Comment:
Same here - do we really need to sort work items by whether the associated task is archived? This is not something the users of work items would be interested in

Reply:
removed